examples/ComplexExample/app/reducers/navigation.js

```diff
@@ -0,0 +1,5 @@
+import RootNavigator from "../navigators/root";
+
+export default function navigation(state, action) {
+  return RootNavigator.router.getStateForAction(action, state);
```

Comment:
Since https://github.com/react-community/react-navigation/pull/431 was merged, what is the recommendation now ?
Like the following: 
```
return RootNavigator.router.getStateForAction(action, state) || state;
```
Or will the bug be fixed internally ?